src/systems/filecoin_mining/storage_mining/constants.go

```diff
@@ -7,3 +7,4 @@ const MAX_PROVE_COMMIT_SECTOR_PERIOD = block.ChainEpoch(3)    // placeholder
 const MAX_SURPRISE_POST_RESPONSE_PERIOD = block.ChainEpoch(4) // placeholder
 const POST_CHALLENGE_TIME = block.ChainEpoch(1)               // placeholder
 const PROVING_PERIOD = block.ChainEpoch(2)                    // placeholder
+const CHALLENGE_FREQUENCY = 2
```

Comment:
Document this: what challenge, in what units?

Reply:
documented in the election_post.md file already but changed to more explicit variable names :)